docs/api/web-view-tag.md

```diff
@@ -128,6 +128,23 @@ Sets the user agent for the guest page before the page is navigated to. Once the
 
 If "on", the guest page will have web security disabled.
 
+### partition
+
+```html
+<webview src="https://github.com" partition="persist:github"></webview>
+<webview src="http://electron.atom.io" partition="electron"></webview>
+```
+
+Sets the storage partition used by the `webview`. If the storage partition ID starts with `persist:`,
+the `webview` will use a persistent storage partition available to all `webview` in the app with
+the same storage partition ID. if there is no `persist:` prefix, the `webview` will
+use an in-memory storage partition. By assigning the same partition ID, multiple `webview`
+can share the same storage partition.
```

Comment:
We should mention that when `partition` is not set webview will reuse current session.